requirements-test.txt

```diff
@@ -0,0 +1,3 @@
+flake8
```

Comment:
this is super opinionated and doesn't change anything functionally, but i believe `test-requirements.txt` is the usual name of the file, not `requirements-test.txt`. I can't find any docs on what pip handles but I've always used `test-requirements.txt` for my python projects.

a quick and dirty search on github: 
* https://github.com/search?q=test-requirements.txt&type=Code (47k hits)
vs
* https://github.com/search?q=requirements-test.txt&type=Code (5k hits)

Reply:
I think I had seen this page (https://pypi.org/project/requirements-test.txt/) when naming that file and assumed it was the standard, I'll update it to match the more common naming.

Reply:
https://pypi.org/project/requirements-test.txt/ was made (and last updated in 2016) by the pypa team to prevent squatters trying to install naughty python code on an easy typo.